packages/inter-protocol/src/psm/psm.js

```diff
@@ -290,6 +292,8 @@ export const start = async (zcf, privateArgs, baggage) => {
         M.splitRecord({
           give: { In: stableAmountShape },
           want: M.or({ Out: anchorAmountShape }, {}),
+          multiples: 1n,
+          exit: M.any(),
         }),
```

Comment:
This now represents all the fields of the record, not just a subset. Thus, the `M.splitRecord` should be removed and the record pattern should be the proposalShape directly.